lib/ffmpeg/transcoder.rb

```diff
@@ -204,6 +210,14 @@ def transcode_movie
           raise Error, "Process hung. Full output: #{@output}"
         end
       end
+      if File.exist?(temp_output_file)
+        FileUtils.cp(temp_output_file, @output_file)
+        FileUtils.rm_rf(temp_output_file)
+      end
+    ensure
+      @movie.interim_paths.each do |path|
+        FileUtils.rm_rf(path) if path.start_with?("#{TEMP_DIR}")
```

Comment:
Nit: no need to be a string interpolation
```suggestion
        FileUtils.rm_rf(path) if path.start_with?(TEMP_DIR)
```